boards/m5stamp-c3.json

```diff
@@ -0,0 +1,33 @@
+{
+  "build": {
+    "arduino":{
+      "ldscript": "esp32c3_out.ld"
+    },
+    "core": "esp32",
+    "f_cpu": "160000000L",
+    "f_flash": "40000000L",
+    "flash_mode": "dio",
+    "mcu": "esp32c3",
+    "variant": "esp32c3"
+  },
+  "connectivity": [
+    "wifi",
+    "bluetooth"
+  ],
+  "debug": {
+    "openocd_target": "esp32c3.cfg"
+  },
+  "frameworks": [
+    "espidf"
+  ],
```

Comment:
Is it possible for this to support Arduino? This board supposedly does.

Reply:
I'll play around with that, we'll see.

Reply:
https://github.com/jcgruenhage/platform-espressif32/commit/34858dfb5497ab98865a1d6f59b46f6dbf035a7f works, but it's not based on develop, but on feature/arduino-upstream, (which seems not to have been merged since I started working on this back in February).

Reply:
Hm, thanks for checking on this. I may just stick with the esp framework then.